jetty-util/src/main/java/org/eclipse/jetty/util/log/JavaUtilLog.java

```diff
@@ -65,8 +65,8 @@
 {
     private static final String THIS_CLASS = JavaUtilLog.class.getName();
```

Comment:
This constant is ever used once, so it can just be inlined.

Reply:
Done.